apps/hasura.planx.uk/migrations/default/1764778977255_v2_filter_delete_events/up.sql

```diff
@@ -0,0 +1,158 @@
+CREATE
+OR REPLACE VIEW "public"."submission_services_log" AS WITH payments AS (
+  SELECT
+    ps.session_id,
+    ps.payment_id AS event_id,
+    'Pay' :: text AS event_type,
+    initcap(ps.status) AS status,
+    jsonb_build_object(
+      'status',
+      ps.status,
+      'description',
+      pse.comment,
+      'govuk_pay_reference',
+      ps.payment_id
+    ) AS response,
+    ps.created_at,
+    false AS retry
+  FROM
+    (
+      payment_status ps
+      LEFT JOIN payment_status_enum pse ON ((pse.value = ps.status))
+    )
+  WHERE
+    (
+      (ps.status <> 'created' :: text)
+      AND (
+        ps.created_at >= '2024-01-01 00:00:00+00' :: timestamp with time zone
+      )
+    )
+),
+retries AS (
+  SELECT
+    hdb_scheduled_event_invocation_logs.id
+  FROM
+    hdb_catalog.hdb_scheduled_event_invocation_logs
+  WHERE
+    (
+      (
+        hdb_scheduled_event_invocation_logs.event_id,
+        hdb_scheduled_event_invocation_logs.created_at
+      ) IN (
+        SELECT
+          seil.event_id,
+          max(seil.created_at) AS max
+        FROM
+          (
+            hdb_catalog.hdb_scheduled_event_invocation_logs seil
+            LEFT JOIN hdb_catalog.hdb_scheduled_events se ON ((se.id = seil.event_id))
+          )
+        WHERE
+          (se.tries > 1)
+        GROUP BY
+          seil.event_id
+      )
+    )
+),
+submissions AS (
+  SELECT
+    (
+      (
+        (
+          (seil.request -> 'payload' :: text) -> 'payload' :: text
+        ) ->> 'sessionId' :: text
+      )
+    ) :: uuid AS session_id,
+    se.id AS event_id,
+    CASE
+      WHEN ((se.webhook_conf) :: text ~~ '%bops%' :: text) THEN 'Submit to BOPS' :: text
+      WHEN ((se.webhook_conf) :: text ~~ '%uniform%' :: text) THEN 'Submit to Uniform' :: text
+      WHEN (
+        (se.webhook_conf) :: text ~~ '%email-submission%' :: text
+      ) THEN 'Send to email' :: text
+      WHEN (
+        (se.webhook_conf) :: text ~~ '%upload-submission%' :: text
+      ) THEN 'Upload to AWS S3' :: text
+      WHEN ((se.webhook_conf) :: text ~~ '%idox%' :: text) THEN 'Submit to Idox Nexus' :: text
+      ELSE (se.webhook_conf) :: text
+    END AS event_type,
+    CASE
+      WHEN (seil.status = 200) THEN 'Success' :: text
+      ELSE format('Failed (%s)' :: text, seil.status)
+    END AS status,
+    (seil.response) :: jsonb AS response,
+    seil.created_at,
+    (
+      EXISTS (
+        SELECT
+          1
+        FROM
+          retries r
+        WHERE
+          (r.id = seil.id)
+      )
+    ) AS retry
+  FROM
+    (
+      hdb_catalog.hdb_scheduled_events se
+      LEFT JOIN hdb_catalog.hdb_scheduled_event_invocation_logs seil ON ((seil.event_id = se.id))
+    )
+  WHERE
+    (
+      -- Exclude email events
+      ((se.webhook_conf) :: text !~~ '%/send-email%' :: text)
+      AND (
+      -- Exclude delete events
+        (se.webhook_conf) :: text !~~ '%/delete-session%' :: text
```

Comment:
This is the only change - before it was `'%delete-session/%'`

This now matches the `hdb_catalog.hdb_scheduled_events` table - 

<img width="721" height="210" alt="Image" src="https://github.com/user-attachments/assets/b6857a64-abe1-418b-af92-505e71cff0c3" />